diff -ruN solid-5.57.0.orig/src/solid/devices/backends/iokit/cfhelper.cpp solid-5.57.0/src/solid/devices/backends/iokit/cfhelper.cpp
--- solid-5.57.0.orig/src/solid/devices/backends/iokit/cfhelper.cpp	2019-04-07 09:34:59.000000000 +0200
+++ solid-5.57.0/src/solid/devices/backends/iokit/cfhelper.cpp	2019-04-21 22:19:48.561728978 +0200
@@ -94,7 +94,7 @@
 
     if (typeId == CFDateGetTypeID()) {
         QDateTime dt;
-        dt.setTime_t(uint(kCFAbsoluteTimeIntervalSince1970));
+        dt.setSecsSinceEpoch(qint64(kCFAbsoluteTimeIntervalSince1970));
         return dt.addSecs(int(CFDateGetAbsoluteTime(static_cast<const CFDateRef>(obj))));
     }
 
