diff -ruN qtbase-everywhere-src-5.12.3.orig/src/src.pro qtbase-everywhere-src-5.12.3/src/src.pro
--- qtbase-everywhere-src-5.12.3.orig/src/src.pro	2019-04-09 11:51:26.000000000 +0200
+++ qtbase-everywhere-src-5.12.3/src/src.pro	2019-05-21 16:01:25.717680488 +0200
@@ -212,9 +212,11 @@
         SUBDIRS += src_3rdparty_freetype
         src_platformsupport.depends += src_3rdparty_freetype
     }
-    SUBDIRS += src_tools_qvkgen
-    src_gui.depends += src_tools_qvkgen
-    TOOLS += src_tools_qvkgen
+    qtConfig(vulkan) {
+        SUBDIRS += src_tools_qvkgen
+        src_gui.depends += src_tools_qvkgen
+        TOOLS += src_tools_qvkgen
+    }
     SUBDIRS += src_gui src_platformsupport src_platformheaders
     qtConfig(opengl): SUBDIRS += src_openglextensions
     src_plugins.depends += src_gui src_platformsupport src_platformheaders
