diff -ruN giflib-5.1.9.orig/Makefile giflib-5.1.9/Makefile
--- giflib-5.1.9.orig/Makefile	2019-03-28 19:57:23.000000000 +0100
+++ giflib-5.1.9/Makefile	2019-05-14 21:29:16.000000000 +0200
@@ -62,19 +62,19 @@
 
 LDLIBS=libgif.a -lm
 
-all: libgif.so libgif.a libutil.so libutil.a $(UTILS)
+all: libgif.$(LIBMAJOR).dylib libgif.a libutil.$(LIBMAJOR).dylib libutil.a $(UTILS)
 	$(MAKE) -C doc
 
 $(UTILS):: libgif.a libutil.a
 
-libgif.so: $(OBJECTS) $(HEADERS)
-	$(CC) $(CFLAGS) -shared $(LDFLAGS) -Wl,-soname -Wl,libgif.so.$(LIBMAJOR) -o libgif.so $(OBJECTS)
+libgif.$(LIBMAJOR).dylib: $(OBJECTS) $(HEADERS)
+	$(CC) $(CFLAGS) -dynamiclib -current_version $(LIBVER) $(OBJECTS) -o libgif.$(LIBMAJOR).dylib
 
 libgif.a: $(OBJECTS) $(HEADERS)
 	$(AR) rcs libgif.a $(OBJECTS)
 
-libutil.so: $(UOBJECTS) $(UHEADERS)
-	$(CC) $(CFLAGS) -shared $(LDFLAGS) -Wl,-soname -Wl,libutil.so.$(LIBMAJOR) -o libutil.so $(UOBJECTS)
+libutil.$(LIBMAJOR).dylib: $(UOBJECTS) $(UHEADERS)
+	$(CC) $(CFLAGS) libgif.$(LIBMAJOR).dylib -dynamiclib -current_version $(LIBVER) $(UOBJECTS) -o libutil.$(LIBMAJOR).dylib
 
 libutil.a: $(UOBJECTS) $(UHEADERS)
 	$(AR) rcs libutil.a $(UOBJECTS)
@@ -100,9 +100,9 @@
 install-lib:
 	$(INSTALL) -d "$(DESTDIR)$(LIBDIR)"
 	$(INSTALL) -m 644 libgif.a "$(DESTDIR)$(LIBDIR)/libgif.a"
-	$(INSTALL) -m 755 libgif.so "$(DESTDIR)$(LIBDIR)/libgif.so.$(LIBVER)"
-	ln -sf libgif.so.$(LIBVER) "$(DESTDIR)$(LIBDIR)/libgif.so.$(LIBMAJOR)"
-	ln -sf libgif.so.$(LIBMAJOR) "$(DESTDIR)$(LIBDIR)/libgif.so"
+	$(INSTALL) -m 755 libgif.$(LIBMAJOR).dylib "$(DESTDIR)$(LIBDIR)/libgif.$(LIBVER).dylib"
+	ln -sf libgif.$(LIBVER).dylib "$(DESTDIR)$(LIBDIR)/libgif.$(LIBMAJOR).dylib"
+	ln -sf libgif.$(LIBMAJOR).dylib "$(DESTDIR)$(LIBDIR)/libgif.dylib"
 install-man:
 	$(INSTALL) -d "$(DESTDIR)$(MANDIR)/man1"
 	$(INSTALL) -m 644 doc/*.1 "$(DESTDIR)$(MANDIR)/man1"
