diff -ruN OpenSP-1.5.2.orig/Makefile.am OpenSP-1.5.2/Makefile.am
--- OpenSP-1.5.2.orig/Makefile.am	2005-12-23 15:14:27.000000000 +0100
+++ OpenSP-1.5.2/Makefile.am	2019-04-28 12:32:00.807870300 +0200
@@ -5,7 +5,7 @@
 SUBDIRS = pubtext unicode doc \
 	include generic \
 	intl po \
-	lib nsgmls spent spam sgmlnorm sx spcat \
+    lib \
 	tests @DOCSRC@
 
 pkginclude_HEADERS = config.h
diff -ruN OpenSP-1.5.2.orig/Makefile.in OpenSP-1.5.2/Makefile.in
--- OpenSP-1.5.2.orig/Makefile.in	2005-12-23 15:16:01.000000000 +0100
+++ OpenSP-1.5.2/Makefile.in	2019-04-28 12:31:52.399455500 +0200
@@ -250,7 +250,7 @@
 SUBDIRS = pubtext unicode doc \
 	include generic \
 	intl po \
-	lib nsgmls spent spam sgmlnorm sx spcat \
+    lib \
 	tests @DOCSRC@
 
 pkginclude_HEADERS = config.h
