diff -ruN libxml2-2.9.10.orig/xmlreader.c libxml2-2.9.10/xmlreader.c
--- libxml2-2.9.10.orig/xmlreader.c	2019-10-22 20:46:01 +0000
+++ libxml2-2.9.10/xmlreader.c	2020-04-25 11:23:19 +0000
@@ -2281,6 +2281,7 @@
     if (reader->ctxt != NULL) {
         if (reader->dict == reader->ctxt->dict)
 	    reader->dict = NULL;
+#ifdef LIBXML_VALID_ENABLED
 	if ((reader->ctxt->vctxt.vstateTab != NULL) &&
 	    (reader->ctxt->vctxt.vstateMax > 0)){
             while (reader->ctxt->vctxt.vstateNr > 0)
@@ -2289,6 +2290,7 @@
 	    reader->ctxt->vctxt.vstateTab = NULL;
 	    reader->ctxt->vctxt.vstateMax = 0;
 	}
+#endif /* LIBXML_VALID_ENABLED */
 	if (reader->ctxt->myDoc != NULL) {
 	    if (reader->preserve == 0)
 		xmlTextReaderFreeDoc(reader, reader->ctxt->myDoc);
