diff -ruN postgresql-11.3.orig/src/bin/Makefile postgresql-11.3/src/bin/Makefile
--- postgresql-11.3.orig/src/bin/Makefile	2019-05-06 22:46:18.000000000 +0200
+++ postgresql-11.3/src/bin/Makefile	2019-06-03 20:31:42.617858300 +0200
@@ -15,28 +15,14 @@
 
 SUBDIRS = \
 	initdb \
-	pg_archivecleanup \
-	pg_basebackup \
-	pg_config \
-	pg_controldata \
-	pg_ctl \
 	pg_dump \
-	pg_resetwal \
-	pg_rewind \
-	pg_test_fsync \
-	pg_test_timing \
-	pg_upgrade \
-	pg_verify_checksums \
-	pg_waldump \
-	pgbench \
-	psql \
-	scripts
+	psql
 
-ifeq ($(PORTNAME), win32)
-SUBDIRS += pgevent
-else
-ALWAYS_SUBDIRS += pgevent
-endif
+#ifeq ($(PORTNAME), win32)
+#SUBDIRS += pgevent
+#else
+#ALWAYS_SUBDIRS += pgevent
+#endif
 
 $(recurse)
 $(recurse_always)
